app/apps/users/models.py

```diff
@@ -49,3 +49,16 @@ def __str__(self):
     def save(self, *args, **kwargs):
         self.username = generate_username(self.email)
         super().save(*args, **kwargs)
+
+
+class SupportContact(models.Model):
+    """
+    Name and phonenumber that users can call if they need support
+    Needs to be a model so that I can be edited in the django admin
```

Comment:
Typo: that **I** can be edited -> should be **it** can be edited
phonenumber -> phone number

Reply:
Scherp!